scripts/setProofRequest.ts

```diff
@@ -0,0 +1,199 @@
+import hre, { ethers } from "hardhat";
+import { packV3ValidatorParams, packValidatorParams } from "../test/utils/validator-pack-utils";
+import { calculateQueryHashV2, calculateQueryHashV3 } from "../test/utils/query-hash-utils";
+import { Blockchain, DidMethod, NetworkId, DID } from "@iden3/js-iden3-core";
+import { buildVerifierId } from "./deployCrossChainVerifierWithRequests";
+import { getConfig } from "../helpers/helperUtils";
+
+async function main() {
+  const MTP_V2_CIRCUIT_NAME = "credentialAtomicQueryMTPV2OnChain";
+  const SIG_V2_CIRCUIT_NAME = "credentialAtomicQuerySigV2OnChain";
+  const V3_CIRCUIT_NAME = "credentialAtomicQueryV3OnChain-beta.1";
+
+  const circuitName: string = V3_CIRCUIT_NAME; // TODO put your circuit here;
+  const requestId = 117; // TODO put your request here;
+  const allowedIssuers = []; // TODO put your allowed issuers here
+
+  const chainId = hre.network.config.chainId;
+  const network = hre.network.name;
+
+  const methodId = "ade09fcd";
+
+  const config = getConfig();
+  if (!ethers.isAddress(config.universalVerifierContractAddress)) {
+    throw new Error("UNIVERSAL_VERIFIER_CONTRACT_ADDRESS is not set");
+  }
+
+  const verifier = await ethers.getContractAt(
+    "UniversalVerifier",
+    config.universalVerifierContractAddress,
+  );
+
+  const verifierId = buildVerifierId(await verifier.getAddress(), {
+    blockchain: Blockchain.Polygon,
+    networkId: NetworkId.Amoy,
+    method: DidMethod.Iden3,
+  });
+
+  const Operators = {
+    NOOP: 0, // No operation, skip query verification in circuit
+    EQ: 1, // equal
+    LT: 2, // less than
+    GT: 3, // greater than
+    IN: 4, // in
+    NIN: 5, // not in
+    NE: 6, // not equal
```

Comment:
there are missing operators (LTE, GTE, BETWEEN...)

Reply:
Now imported from js-sdk to cover all of them
```
import { Operators } from "@0xpolygonid/js-sdk";
```